src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/NumberHandlingTests.cs

```diff
@@ -1508,39 +1508,30 @@ public class ClassWith_NumberHandlingOn_ObjectProperty
         [ActiveIssue("Need to tweak number handling option registration following code-gen support.")]
         public static void Attribute_NotAllowed_On_Property_WithCustomConverter()
```

Comment:
Test name should probably be updated to reflect the updated semantics.

Reply:
Yep thanks. I may commit this PR and fix in a follow-up.

Reply:
Thanks